BackendServer/db/users.js

```diff
@@ -0,0 +1,52 @@
+var AWS = require( 'aws-sdk' );
```

Comment:
You are using const, which is ES6 syntax. If you are using ES6 syntax, change all of your var variable declarations to let